### PR TITLE
adding first draft of the login page

### PR DESCRIPTION
**Pull request by sophiecs333**
Hey!
Let's review the login page together.
I don't manage to center it properly on the page and to style the two "signup" and "forgot your password" texts.
Of course we can update the style as well ;)

### CODE REVIEW

File: app/assets/stylesheets/pages/_sign-in.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.container {
```
Comment (dumboluzz) [inline app/assets/stylesheets/pages/_sign-in.scss:1]: `container` is an important bootstrap class. Please don't target it. You can change the name of your custom class.

File: app/assets/stylesheets/pages/_sign-in.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  background-color: #92B4EC;
+  display: flex;
+  flex-direction: column;
+  width: 30%;
```
Comment (dumboluzz) [inline app/assets/stylesheets/pages/_sign-in.scss:5]: make the width of the login card responsive to the screen size

File: app/assets/stylesheets/pages/_sign-in.scss [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  padding: 40px 50px;
+  margin-top: 50px;
```
Comment (dumboluzz) [inline app/assets/stylesheets/pages/_sign-in.scss:7]: `margin-top` should go below `margin`

File: app/assets/stylesheets/pages/_sign-in.scss [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  margin: auto;
```
Comment (dumboluzz) [inline app/assets/stylesheets/pages/_sign-in.scss:8]: `margin: 0 auto;` is properly the thing that you want.

**Review by dumboluzz — CHANGES_REQUESTED** [review]
try to use bootstrap to layout the page and only use custom css to add custom styling (eg. color and fonts)

**Comment by sophiecs333** [comment]
Heyyyy please review again thanks!

**Pull request merged**